dracut.sh

```diff
@@ -1685,7 +1696,7 @@ for line in "${fstab_lines[@]}"; do
             push_host_devs "$mp"
         done
     fi
-    push_host_devs "$dev"
+    push_user_devs "$dev"
     host_fs_types["$dev"]="$3"
```

Comment:
If you push these to `user_devs` rather than `host_devs`, these devices won't be taken into account in `for_each_host_dev_and_slaves_all()`, which might lead to missing modules in hostonly mode. Or am I missing something?

Reply:
No, you stand correct, I was missing that. Thanks.